core/eolearn/core/eodata_io.py

```diff
@@ -99,6 +99,14 @@ def iterate_features(self) -> Iterator[tuple[tuple[FeatureType, str], str]]:
                 yield (ftype, fname), path
 
 
+@dataclass
+class InferredSelection:
+    """Temporal selection that was inferred from timestamps. Also carries info on how to initialize Zarr if needed."""
```

Comment:
> Temporal selection that was inferred from timestamps.

-> 

Temporal selection that was inferred from existing(?) timestamps.

Reply:
Changed the name and the description. At some point this was obtained just when the selection is inferred, but now even predefined selections (such as `slice(1,2)`) are transformed into such objects, so `TemporalSelection` is a more fitting name

Reply:
Sounds much better, yes.